frontend/src/app/starters/InterestsStarter.tsx

```diff
@@ -1,20 +1,40 @@
 import React from 'react'
 import './InterestsStarter.css'
 import { ConversationStarterProps } from './ConversationStarter'
+import { pluralize } from '../utils'
+import { CommunityData } from '../../config/communities'
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 interface InterestsStarterProps {
-    // TODO: Fill out the props for your component
+    chatInterests: Array<string>
+    communityData: CommunityData
 }
 
-// TODO: Implement your component
-// eslint-disable-next-line no-empty-pattern
-const InterestsStarterInner = ({}: InterestsStarterProps) => {
-    return <p>You have some interests in common!</p>
+const InterestsStarterInner = ({ chatInterests, communityData }: InterestsStarterProps) => {
```

Comment:
Great work, Liliana! This looks very polished and is working in the app when I tested it out.

Reply:
Thank you!!! :D